pkg/web/apps/apps_psql_repository.go

```diff
@@ -11,6 +11,10 @@ type (
 	// Service defines the interface methods to be used
 	AppPostgreSQLRepository interface {
 		GetApps() (*[]models.App, error)
+		GetAppByID(ID string) (*models.App, error)
```

Comment:
The `AppPostgreSQLRepository` and `PostgreSQLRepository` interfaces in thsi file are not needed. The `NewPostgreSQLRepository` func uses the generic `Repository` interface.

Reply:
@davidffrench unfurtunilly, they are needed in order to follow the good practices and ensure what method or not it should implement and how as in order to create the mocks.

Reply:
Following some refs:

- https://softwareengineering.stackexchange.com/questions/159813/do-i-need-to-use-an-interface-when-only-one-class-will-ever-implement-it

Reply:
Hi @davidffrench it was not added here and I catcher what you mean. Fixing it.